src/common/constants.ts

```diff
@@ -163,7 +163,7 @@ export const live_chat_key = 'live_chat_redirection'
 export const reddit_url = 'https://www.reddit.com/user/Deriv_official/'
 export const telegram_url = 'https://t.me/derivdotcomofficial'
 export const youtube_url = 'https://www.youtube.com/@deriv'
-export const loss_percent = 72
+export const fallback_loss_percent = 80
```

Comment:
We can't have fallback loss percentage
```suggestion
export const fallback_loss_percent = 80
```

Reply:
thanks, fixed